x/ibc-transfer/keeper/relay.go

```diff
@@ -59,69 +61,54 @@ func (k Keeper) createOutgoingPacket(
 	seq uint64,
 	sourcePort, sourceChannel,
 	destinationPort, destinationChannel string,
-	amount sdk.Coins,
+	amount sdk.Coin,
 	sender sdk.AccAddress,
 	receiver string,
+	source bool,
 	timeoutHeight, timeoutTimestamp uint64,
 ) error {
 	channelCap, ok := k.scopedKeeper.GetCapability(ctx, host.ChannelCapabilityPath(sourcePort, sourceChannel))
 	if !ok {
 		return sdkerrors.Wrap(channeltypes.ErrChannelCapabilityNotFound, "module does not own channel capability")
 	}
-	// NOTE:
-	// - Coins transferred from the destination chain should have their denomination
-	// prefixed with source port and channel IDs.
-	// - Coins transferred from the source chain can have their denomination
-	// clear from prefixes when transferred to the escrow account (i.e when they are
-	// locked) BUT MUST have the destination port and channel ID when constructing
-	// the packet data.
-	if len(amount) != 1 {
-		return sdkerrors.Wrapf(types.ErrOnlyOneDenomAllowed, "%d denoms included", len(amount))
-	}
-
-	prefix := types.GetDenomPrefix(destinationPort, destinationChannel)
-	source := strings.HasPrefix(amount[0].Denom, prefix)
+	// NOTE: When the sender chain is acting as the source zone, the coins will
+	// be constructed with the destination port and channel ID prefixed when
+	// represented in the fungible token packet data. When being escrowed these
+	// coins will not have the prefix.
```

Comment:
```suggestion
	// represented in the fungible token packet data. However the coins
	// will be escrowed on the source chain without the prefix
```

This documentation is clearer to me, I got confused for a bit on the current wording